core-bundle/contao/config/default.php

```diff
@@ -100,7 +100,6 @@
 $GLOBALS['TL_CONFIG']['backendTheme']         = 'flexible';
 $GLOBALS['TL_CONFIG']['doNotCollapse']        = false;
 $GLOBALS['TL_CONFIG']['minPasswordLength']    = 8;
-$GLOBALS['TL_CONFIG']['disableCron']          = false;
```

Comment:
This config is obsolete and not needed anymore. We have a zero config approach. Contao automatically detects if you have a real cron, otherwise you get the poor man's fallback.